api.planx.uk/modules/pay/service/inviteToPay/sendConfirmationEmail.ts

```diff
@@ -22,16 +22,7 @@ export async function sendAgentAndPayeeConfirmationEmail(sessionId: string) {
 }
 
 type PayeeAndAgentEmailData = {
-  personalisation: {
-    emailReplyToId: string;
-    helpEmail: string;
-    helpOpeningHours: string;
-    helpPhone: string;
-    serviceName: string;
-    payeeName: string;
-    address: string;
-    applicantName: string;
-  };
+  personalisation: Omit<AgentAndPayeeSubmissionNotifyPersonalisation, "projectType">;
```

Comment:
Here's the template, this type and query now match it correctly - 

<img width="744" alt="image" src="https://github.com/user-attachments/assets/6fa025c3-975d-4b57-8a39-e12616e332c5">

`projectTypes` is converted to `projectType` in the next function call.